docs/podman-inspect.1.md

```diff
@@ -19,8 +19,13 @@ Return JSON for the specified type.  Type can be 'container', 'image' or 'all' (
 
 **--format, -f="FORMAT"**
 
-Format the output using the given Go template.
-The keys of the returned JSON can be used as the values for the --format flag (see examples below).
+Format the output using the given Go template. The keys of the returned JSON can be used as the values for
+the --format flag (see examples below).
+
+You may also pass a format of **kube** to generate kubernetes YAML that describes the current container. This
+YAML can then be used to recreate a container in a runtime that understands kubernetes. When using the **kube**
+output format, each container is described inside the context of a simple Pod.  This function is still very
```

Comment:
I would just leave out the comment about heavy development.  I worry about this comment being left there forever.

We should point out that there are two special formats "json", and "kube" as well as other format strings.

Reply:
I deleted the kube part.  The json is not really applicable because in inspect we always output json or use the go templating.